src/_samples/index.md

```diff
@@ -126,23 +130,26 @@ including how the documentation tooling works.
 
 ## Imports
 
+<?code-excerpt "test/samples_test.dart (import)" replace="/'.*?' as \w+/'path\/to\/my_other_file.dart'/g"?>
 {% prettify dart %}
 // Importing core libraries
 import 'dart:async';
 import 'dart:math';
 
 // Importing libraries from external packages
-import 'package:angular2/angular2.dart';
+import 'package:test/test.dart';
```

Comment:
I figured that external package would do and so went with `test`.  Avoiding angular here means that we avoid the delays on precompiling it when `pub get/upgrade` is run.